contributors/devel/sig-testing/writing-good-e2e-tests.md

```diff
@@ -38,17 +52,288 @@ like the following generates rather useless errors:
 Expect(err).NotTo(HaveOccurred())
 ```
 
-Rather
+Errors returned by client-go can be very detailed. A better way to test for
+errors is with `framework.ExpectNoError` because it logs the full error and
```

Comment:
I see , this seems to be the only helper not wrapping gomega
https://github.com/kubernetes/kubernetes/pull/115961